### PR TITLE
Onboarding: Successful Anserini Reproduction

### PR DESCRIPTION
**Pull request by Linsen-gao-457**
Hi! Thank you for your patience! It's my first time to use github for collaboration... so i have made some mistakes. I hope this won’t cause any inconvenience. And i will show the details below, if there are any other peoblems, plz let me know. Thanks again!

Begin my journey:
ECEUbnuntu (limitted storage)
Operating System: Ubuntu 20.04.6 LTS
Python Version : 3.8.10
Maven Version : 3.8.1
Memory: 125 GiB(only 8GB for ECE students)
Outcome: All steps completed successfully

Second project:
My window computer
Operating System: Windows 10 Home Chinese Version
Java Version: 21.0.4 2024-07-16 LTS
Apache Maven Version: 3.9.9
Python Version : 3.8.10
Outcome: All steps completed successfully

Issues Encountered:
2 problems - mvn clean package
1. Windows Virtual Memory Allocation Issue:
High memory usage during inverted index construction caused system memory exhaustion.
Solution: Manually increased Windows virtual memory allocation to approximately 200 GB to ensure the indexing process completed smoothly.
2. I use mvn clean package -DskipTests instead of -Dmaven.test.skip=true, but mvn clean package "-Dmaven.test.skip=true" can work to install Anserini. The reason -DskipTests and -Dmaven.test.skip=true work seamlessly while -Dmaven.test.skip=true may fail in PowerShell is due to how Maven and the terminal handle arguments:

Another problem - TREC Tool Incompatibility on Windows: The default trec_eval tool does not support native Windows execution.
Solution: Recompiled trec_eval using MSYS2 to successfully execute the tool

### CODE REVIEW

File: docs/experiments-msmarco-passage.md
Comment (lintool) [inline docs/experiments-msmarco-passage.md:545]: Please change @Forrest to @Linsen-gao-457 , your actual user name?

**Review by lintool — CHANGES_REQUESTED** [review]

File: docs/experiments-msmarco-passage.md
Comment (Linsen-gao-457) [reply docs/experiments-msmarco-passage.md:545]: OK!

Comment (Linsen-gao-457) [reply docs/experiments-msmarco-passage.md]: > Please change @forrest to @Linsen-gao-457 , your actual user name?

Already update! Is that ok now? or i need to do a new pull request

Comment (lintool) [reply docs/experiments-msmarco-passage.md]: all good, waiting for tests to pass before I merge

Comment (Linsen-gao-457) [reply docs/experiments-msmarco-passage.md]: "Got it! thank you very much!!

**Review by lintool — APPROVED** [review]

**Pull request merged**
